src/Suave/Http.fs

```diff
@@ -806,3 +806,12 @@ module Http =
 
     [<System.Obsolete("Use authenticateBasic")>]
     let authenticate_basic f ctx = authenticateBasic f ctx
+
+  module Control =
+
+    let CLOSE (ctx: HttpContext) = 
```

Comment:
In what other control contexts that need to be user-accessible, than WebSockets can this be used?